webrtc.html

```diff
@@ -1714,14 +1726,27 @@ <h4>Set the RTCSessionDescription</h4>
                             "applying-a-remote-desc">[[!JSEP]]</span>.</p>
                           </li>
                           <li>
-                            <p>If <var>direction</var> is <code>"sendrecv"</code> or
-                            <code>"recvonly"</code>, and
-                            <var>transceiver</var>'s
+                            <p>If <var>direction</var> is <code>"sendrecv"</code>
+                            or <code>"recvonly"</code>, let <var>msids</var> be a
+                            list of the MSIDs that the media description indicates
+                            <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\ReceiverTrack]]</a>
+                            is to be associated with. Otherwise, let
+                            <var>msids</var> be an empty list.</p>
+                          </li>
+                          <li>
+                            <p><a>Set the associated remote streams</a> given
+                            <var>transceiver</var>.<a>[[\Receiver]]</a>,
+                            <var>msids</var>, <var>addList</var>, and
+                            <var>removeList</var>.</p>
+                          </li>
+                          <li>
+                            <p>If the previous step increased the length of
+                            <var>addList</var>, or if <var>transceiver</var>'s
```

Comment:
If "ontrack" is the catch-all "streams changed" event, then we need to process the addition of a remote track even if only the removeList increased in length.